targets/targets.json

```diff
@@ -151,7 +151,7 @@
         "inherits": ["LPCTarget"],
         "core": "Cortex-M0",
         "default_toolchain": "uARM",
-        "extra_labels": ["NXP", "LPC11UXX", "MCU_LPC11U35_501"],
+        "extra_labels": ["NXP", "LPC11UXX", "MCU_LPC11U35_501", "LPC11U35_501"],
```

Comment:
Instead, you could have the `XADOW_M0` inherit from the `LPC11U35_501` and modify the target as needed, without adding the `release_version` key.